src/stackedtime/sim_nr.jl

```diff
@@ -78,3 +107,23 @@ Solve the simulation problem.
     return false
 end
 
+"""
+
+"""
+function schedule_λ_default(λ::Float64, it::Int, nFx::Float64, Δx::AbstractArray{Float64})
+    nΔx = λ * norm(vec(Δx), Inf)
+    # Initialize λ
+    if it == 1 && nFx >= 0.5
+        λ = 0.1250
+    end
+    # Set Δλ
+    (nFx > 1.) ? (Δλ = 0.05) : (Δλ = 0.15)
+    # Start to increase λ by Δλ
+    if nΔx < 0.50
+        λ += Δλ
+    end
+    # λ cannot exceed 1
+    λ = min(λ,1.)
```

Comment:
This can also be put inside the above if statement since lamda shouldn't be above 1 if we didn't change it.